app/views/bgeigie_imports/_static_metadata.html.erb

```diff
@@ -1,28 +1,14 @@
 <div class="span2">
   <dl>
-    <dt>
-      <%= t('.uploaded_by') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.user.name_or_email %>
-    </dd>
-    <dt>
-      <%= t('.filename') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.source.filename %>
-    </dd>
-    <dt>
-      <%= t('.number_of_lines') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.lines_count %>
-    </dd>
-    <dt>
-      <%= t('.number_of_measurements') %>
-    </dt>
-    <dd>
-      <%= @bgeigie_import.processed? ? @bgeigie_import.bgeigie_logs.size : '-' %>
-    </dd>
+    <dt><%= t('.uploaded_by') %></dt>
+    <dd><%= @bgeigie_import.user.name_or_email %></dd>
+    <dt><%= t('.filename') %></dt>
+    <dd><%= @bgeigie_import.source.filename %></dd>
+    <dt><%= t('.number_of_lines') %></dt>
+    <dd><%= @bgeigie_import.lines_count %></dd>
+    <dt><%= t('.number_of_measurements') %></dt>
+    <dd><%= @bgeigie_import.processed? ? @bgeigie_import.bgeigie_logs.size : '-' %></dd>
+    <dt>Highest Measurement</dt>
+    <dd><%= @bgeigie_import.maximum_cpm || '-' %></dd>
```

Comment:
Looks like the show page isn't bad anymore, but could be good to check on [newrelic](https://rpm.newrelic.com/accounts/1057812/applications/10580222/transactions?type=app#id=5b22436f6e74726f6c6c65722f626765696769655f696d706f7274732f73686f77222c22225d) once this is deployed to ensure the new `max()` call doesn't cause undue load.

Reply:
Newrelic is looking clean. And also looks like that page doesn't see massive load anyway, so I'd say we're 👍 

![screen shot 2018-07-12 at 8 22 47](https://user-images.githubusercontent.com/690/42604267-d3e137ce-85ac-11e8-9d01-5f274a3e9d73.png)

Reply:
Thank you for checking out.